backend/package.json

```diff
@@ -28,14 +28,14 @@
     "helmet": "^7.1.0",
     "jsonwebtoken": "^9.0.2",
     "morgan": "^1.10.0",
+    "nodemon": "^3.1.1",
     "per-env": "^1.0.2",
     "pg": "^8.11.5",
     "sequelize": "^6.37.3",
     "sequelize-cli": "^6.6.2"
   },
   "devDependencies": {
     "dotenv-cli": "^7.4.2",
-    "nodemon": "^3.1.0",
```

Comment:
Keep in mind the nodemon should be a developer dependency and not a production dependency